test/e2e/storage/volume_provisioning.go

```diff
@@ -716,6 +790,37 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 			Expect(claim.Status.Phase).To(Equal(v1.ClaimPending))
 		})
 	})
+
+	framework.KubeDescribe("GlusterDynamicProvisioner", func() {
```

Comment:
Need `[Slow]` here.  If all test provisioners (gce, gke, aws?) support native Gluster then no extra tag is needed

Reply:
not planning to mix this with other cloud providers and the operation should be fast enough to complete.